wn/lmf.py

```diff
@@ -482,7 +510,7 @@ def __init__(
         self.requires = requires or []
 
 
-LexicalResource = List[Lexicon]
+LexicalResource = Iterator[Lexicon]
```

Comment:
Why is this change happening (here and throughout)? It seems unrelated to the addition of a progress bar.